### ISSUE TITLE
Notepad++ 7.3.3 double click bug.

### ISSUE THREAD

**Issue by AraHaan** [notepad-plus-plus/notepad-plus-plus#3426]
### Description of the Issue

Sometimes when double clicking a word or something it selects spaces and iother words magically as well. This happens in sometimes rare cases. But bites me in the but frequently oddly.
![ss 2017-06-20 at 06 21 48](https://user-images.githubusercontent.com/15173749/27360305-ef3c297c-55e5-11e7-8746-01384ccf2a5b.png)

### Steps to Reproduce the Issue

1. The steps is unknown to me. But probably is repoduced by running NP++ constantly for hours or days on end without closing. But yeah at the moment I am just not sure exactly how to reprduce it.

### Expected Behavior

Everuything to work like normal.

### Actual Behavior

See above.

### Debug Information

```
Notepad++ v7.3.3   (32-bit)
Build time : Mar  8 2017 - 03:37:37
Path : E:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 7 (64-bit)
Plugins : MenuIcons.dll mimeTools.dll NppConverter.dll nppcrypt.dll NppExec.dll NppExport.dll NppFTP.dll NPPJSONViewer.dll NppSaveAsAdmin.dll PluginManager.dll PyNPP.dll Python Indent.dll PythonScript.dll ReloadButton.dll Translate.dll ZenCoding-Python.dll zoomdisabler.dll 
```

This did rarely happen in earlier versions as well before I updated to 7.2 or even 7.1 I think.
(Not sure if this was fixed yet)

**Comment by dail8859** [comment]
The only thing I can recommend is use version 7.4.2 which has [this](https://github.com/notepad-plus-plus/notepad-plus-plus/commit/c63673a1d8f4774bfcbd01c11d7cb4fc09b9fb2c) commit which fixes a potential buffer overrun of the wordchar list. I was never able to consistently replicate this issue so I can't say for sure it will fix it but it has a good chance of working.

Just FYI there have been numerous reports of this inconsistent behavior: #3137, #3105, #3100, [here](https://notepad-plus-plus.org/community/topic/13568/double-click-selects-into-previous-line), [here](https://notepad-plus-plus.org/community/topic/13509/sci_wordleft-and-sci_wordright-treats-sentences-as-words-in-new-file/2), [here](https://notepad-plus-plus.org/community/topic/13593/inconsistent-double-click-selection-of-quoted-text)

**Comment by dail8859** [comment]
This bug hasn't been reported in a while and hopefully c63673a1d8f4774bfcbd01c11d7cb4fc09b9fb2c was the actual fix to it in v7.4 and afterwards. If you run into this issue again please reopen.
